core/server/services/auth/setup.js

```diff
@@ -107,7 +107,9 @@ async function doSettings(data, settingsAPI) {
     return user;
 }
 
-async function doProduct(data, productsAPI) {
+// Update names for default product and newsletter to site title
+async function doProductAndNewsletter(data, api) {
```

Comment:
Curious: why combine these two concepts rather than handle them separately?

Reply:
I was honestly split between doing them separate or together and tried both, and while not a strong pref but eventually kept together as it seemed to make more sense to keep related default fixture update together under a single fn. Probably could use a fn rename to `doFixtureDefaults` or something, but i am ok to split them up as well if that seems better.

Reply:
> seemed to make more sense to keep related default fixture update together

I'm not sure they are related though apart from performing the same action on them 🤔

It's nabd, but I think as soon as we add another <X> that needs the same treatment we'll want to split them out anyway.